dao/OrderRepository.go

```diff
@@ -0,0 +1,118 @@
+package dao
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+)
+
+type OrderPostgres struct {
+	db *sql.DB
+}
+
+func NewOrderPostgres(db *sql.DB) *OrderPostgres {
+	return &OrderPostgres{db: db}
+}
+
+type Order struct {
+	IdDeliveryService int    `json:"delivery_service_id,omitempty"`
```

Comment:
I honestly don't know if it is easy to fix and how many places like this you already have, but correct naming from english point of view is `DeliveryServiceId`. Feel free to skip this comment and fix it later or even fix it only for newer code, because I understand that such change could be difficult.